tests/test_rbuser_tostring.py

```diff
@@ -0,0 +1,80 @@
+"""RedBrick Test Module; Tests the to_string method of the rbuser module."""
+
+import unittest
+from useradm import rbuser
+
+
+class RBUserTestCase(unittest.TestCase):
+    """Test Case class for to_string method"""
+
+    def setUp(self):
+        """Call before every test case."""
+        self.billy_attr = {'cn': 'Billy Bo',
```

Comment:
That would make very little sense. These tests should represent real life, not a test. Data should not look like a test